base/applications/mspaint/lang/ro-RO.rc

```diff
@@ -43,7 +43,7 @@ BEGIN
         MENUITEM SEPARATOR
         MENUITEM "&Decupează\tCtrl+X", IDM_EDITCUT
         MENUITEM "&Copiază\tCtrl+C", IDM_EDITCOPY
-        MENUITEM "&Lipește\tCtrl+V", IDM_EDITPASTE
+        MENUITEM "Li&pește\tCtrl+V", IDM_EDITPASTE
```

Comment:
I have a hard time trying to understand the reason behind this access key reassignment. I presume that it has something to do with what's in [Microsoft's Guidelines](https://learn.microsoft.com/ro-ro/previous-versions/windows/desktop/dnacc/guidelines-for-keyboard-user-interface-design?redirectedfrom=MSDN#selecting-access-keys) on the matter? There, it says _"Avoid using letters that are difficult to see when they are assigned to access keys. These letters include" "Letters with descenders, such as p, g, q, or y."_ and _"Letters that are only one pixel wide, such as i or l."_ But here, the capital "L" is not that narrow and should have been OK (as the problem with letter is only with its lower case). What you did is to change it to a letter with descender, which the guide is explicitly tells us to avoid!